sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyProperties.java

```diff
@@ -149,21 +149,21 @@ public KeyProperties setNotBefore(OffsetDateTime notBefore) {
      *
      * @return the expires UTC time.
      */
-    public OffsetDateTime getExpires() {
-        if (this.expires == null) {
+    public OffsetDateTime getExpiresOn() {
+        if (this.expiresOn == null) {
```

Comment:
If we want to return null, if it is null, then we don't need this check?

Reply:
Valid point, That's how swagger generated it.
Removed the redundant check.